test/db/formats/mach0/mach0

```diff
@@ -310,3 +310,38 @@ nth paddr      vaddr      bind   type size lib name
 3   0x00007fb0 0x00007fb0 LOCAL  FUNC 0        func.00007fb0
 EOF
 RUN
+
+NAME=mach0 sections
+FILE=bins/mach0/hello-osx-x86_64
+CMDS=iS
+EXPECT=<<EOF
+[Sections]
+
+nth paddr       size vaddr        vsize perm name                     type                     flags
```

Comment:
Unrelated to the PR, but we might need to fix the RzTable API to fix the separator length in the latest column.